### PR TITLE
Lookup by PostCode / Specify Max Dist in KM

### PR DESCRIPTION
**Pull request by 1adam**
Shiny new features; still needs full base_url def'ns

### CODE REVIEW

File: check_all_by_postal_code.py [hunk 1]
```diff
@@ -0,0 +1,108 @@
+import sys
+import json
+import requests
+import haversine
+from xml.dom.minidom import parseString
+from datetime import datetime
+from dateutil import tz
+from haversine import haversine, Unit
```
Comment (1adam) [inline check_all_by_postal_code.py:8]: whoops, "Unit" unneeded without conversion code

Comment (shmick) [reply check_all_by_postal_code.py:8]: Awesome! Thanks.

**Comment by shmick** [comment]
FYI - ```https://geogratis.gc.ca/services/geolocation/en/locate?q=<postalcode>``` returns JSON. No need to mess with xml!

**Comment by shmick** [comment]
Thanks again for your PR.

I've been making some changes and have also Dockerized it as well.

**Pull request merged**